src/vstest.console/CommandLine/AssemblyMetadataProvider.cs

```diff
@@ -103,17 +103,31 @@ private Architecture GetArchitectureFromAssemblyMetadata(string path)
             switch (peReader.PEHeaders.CoffHeader.Machine)
             {
                 case Machine.Amd64:
-                    return Architecture.X64;
                 case Machine.IA64:
```

Comment:
itanium (ia64) is also unsupported obsolete architecture (and it's different than x64). you can delete this line.

Reply:
Thanks, at the moment it's used also in another place in the same file so I'll keep it here for coherence. 
We'll remove in a follow up PR after some analysis cc: @nohwnd